dWorldServer/WorldServer.cpp

```diff
@@ -313,6 +314,9 @@ int main(int argc, char** argv) {
 	uint32_t sqlPingTime = 10 * 60 * currentFramerate; // 10 minutes in frames
 	uint32_t emptyShutdownTime = (cloneID == 0 ? 30 : 5) * 60 * currentFramerate; // 30 minutes for main worlds, 5 for all others.
 
+	// Register slash commands
+	SlashCommandHandler::Startup();
```

Comment:
Genuine like, insane nit pick, this in theory doesn't need to be done on zone 0 worlds servers, so if you are going to make some changes, maybe just put a `if (worldId != 0)` at the start of this line, but really doesn't matter if you don't.

Reply:
I'm going to do a follow up PR and break out SlashCommand.cpp into multiple files. I'll add this in that PR